xml/html_title.xml

```diff
@@ -627,6 +656,17 @@
     <param pos="0" name="hw.device" value="Router"/>
   </fingerprint>
 
+  <fingerprint pattern="^MikroTik SwOS$">
+    <description>MikroTik SwOS switch configuration page</description>
+    <example>MikroTik SwOS</example>
+    <param pos="0" name="os.vendor" value="MikroTik"/>
+    <param pos="0" name="os.device" value="Switch"/>
+    <param pos="0" name="os.product" value="SwOS"/>
+    <param pos="0" name="os.cpe23" value="cpe:/o:mikrotik:swos:-"/>
```

Comment:
Just a note, since this isn't in the NIST database this entry will be destroyed by `update_cpes.py` when we run it after this PR is landed.